service/src/main/java/greencity/mapping/FindAllInfoAboutLocationDtoMapper.java

```diff
@@ -0,0 +1,33 @@
+package greencity.mapping;
+
+import greencity.dto.LocationInfoDto;
+import greencity.dto.LocationTranslationDto;
+import greencity.dto.LocationsDto;
+import greencity.dto.RegionTranslationDto;
+import greencity.entity.user.Region;
+import org.modelmapper.AbstractConverter;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Component
+public class FindAllInfoAboutLocationDtoMapper extends AbstractConverter<Region, LocationInfoDto> {
+    @Override
+    protected LocationInfoDto convert(Region source) {
+        List<LocationsDto> locationsDtoList = source.getLocations().stream()
+            .map(i -> new LocationsDto(i.getId(), i.getLocationStatus().toString(), i.getCoordinates().getLatitude(),
```

Comment:
to create `LocationsDto` it's better to use builder instead of constructor with 4 arguments